docs/manage/fleet/reuse-configuration.md

```diff
@@ -33,33 +33,28 @@ You must be an [organization owner](/manage/manage/rbac/) to create fragments fo
 
 {{< table >}}
 {{% tablestep link="/operate/get-started/setup/" %}}
-**1. Configure one machine**
+**1. In the [Viam app](https://app.viam.com), create a machine** in any {{< glossary_tooltip term_id="location" text="location" >}}.
 
-Start by configuring one of your machines.
+{{% /tablestep %}}
+{{% tablestep link="/operate/get-started/setup/" %}}
+**2. Use the CONFIGURE tab to configure all resources** you want to use on all your machines.
 
-In the [Viam app](https://app.viam.com), use the **CONFIGURE** tab to build a configuration for all resources you want to use on all your machines.
-You can use all available resources in the configuration for your machine except [triggers](/data-ai/data/advanced/alert-data/) which aren't supported.
+All available resources except [triggers](/data-ai/data/advanced/alert-data/) are supported in fragments.
 This means you can also use fragments inside a fragment.
 
-{{<imgproc src="/how-tos/one-to-many/config.png" resize="800x" class="shadow fill" style="width: 400px" declaredimensions=true alt="Configuration builder UI">}}
-
 {{% /tablestep %}}
 {{% tablestep %}}
-**2. Copy the raw JSON**
-
-In your machine's **CONFIGURE** tab, switch to **JSON** and copy the raw JSON configuration object.
+**3. Switch to JSON and copy the raw JSON configuration object**.
 
 {{<imgproc src="/how-tos/one-to-many/raw-json.png" resize="700x" class="shadow fill" style="width: 400px" declaredimensions=true alt="JSON subtab of the CONFIGURE tab">}}
```

Comment:
[opt] Now with the red box, I worry folks might lose the focus on actually copying the JSON text itself. Consider taking a screenshot where the JSON config is selected, ready to copy